commit 2354a83f1a22aab9b0ce480bf3b6df78fb4952b6 (refs/changes/79/2879/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-10-27T22:34:56+00:00 (1 year, 11 months ago)
    
    OS-6425 loader should ignore active multi_vdev_crash_dump feature on zpool
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/boot/sys/boot/zfs/zfsimpl.c b/usr/src/boot/sys/boot/zfs/zfsimpl.c
index f34f88d88e..26654c0469 100644
--- a/usr/src/boot/sys/boot/zfs/zfsimpl.c
+++ b/usr/src/boot/sys/boot/zfs/zfsimpl.c
@@ -59,6 +59,7 @@ static const char *features_for_read[] = {
 	"org.open-zfs:large_blocks",
 	"org.illumos:sha512",
 	"org.zfsonlinux:large_dnode",
+	"com.joyent:multi_vdev_crash_dump",
 	NULL
 };
 
